packages/cli/src/cmds/publish.ts

```diff
@@ -41,8 +41,7 @@ export default class Publish extends YargsCommand {
       CliTelemetry.sendTelemetryEvent(TelemetryEvent.PublishStart);
       result = await activate();
     } else {
-      const rootFolder = answers["folder"] as string;
-      delete answers.folder;
+      const rootFolder = answers.projectPath || ".";
```

Comment:
`|| "."` 

as you mentioned, the logic has already been handled by `argsToInputs` and it already has similar fallback, do we still need it here?

Reply:
Not needed in theory. It is because the type of `projectPath` is `string | undefined`. Changed to `projectPath!` to prevent confusion